lib/kernel-manager.js

```diff
@@ -193,19 +193,20 @@ export default class KernelManager {
 
   getKernelSpecFor(language, callback) {
     if (language == null) {
-      return null;
+      return;
```

Comment:
Unless all the `if(xxx == null)` are fixed, this change will break users of `getKernelSpecFor`